PR/# remove comments in output json str, after js

```diff
@@ -0,0 +1,12 @@
+
+git commit -m "To avoid JSONDecodeError: " -m "Remove comments in output json str, after json value content, maybe start with #, maybe start with //, particularly, it is not inside the string value" -m "Addtionly, if you do not want JSONDecodeError to occur, you can add 'Delete comments in json' after FORMAT_CONSTRAINT in action_node.py"
```

Comment:
These three files appear to be the wrong submissions

Reply:
Yeah, sorry, do you want me to delete it? I need to see how to do.

Reply:
@HuZixia yes. delete PR dir

Reply:
Done～